check_env.py

```diff
@@ -0,0 +1,196 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+# Standard library
+import os
+import json
+import inspect
+import sys
+import types
+import six
+
+from astropy import log as logger
+
+def check_environment(tutorial=None):
+    error=False
+    enter=False
+    warnings=False
+    logger.info("Running Environment Tests...")
+    _orig_path = os.getcwd()
```

Comment:
I think you can make this a little cleaner by not changing directories but instead just construct the full path to `requirements.json` using `tutorial_path`.  Not a big deal but it's often a good idea to not use `chdir` unless you really need to do so.